### PR TITLE
[sentinel_one] Change event.kind and adjust some relevant alert fields

### PR DESCRIPTION
**Pull request by ChriZzn**
## What does this PR do?

Added relevant fields for Alerts and changed the event.kind field to "alert", this is useful to show the Threats from Sentinel One in Kibana under "External Alerts"

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [x] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Author's Checklist

- [ ]

## Related issues

-

### CODE REVIEW

**Comment by andrewkroh** [comment]
Hi @ChriZzn, could you please sign the CLA https://www.elastic.co/contributor-agreement? Thanks.

File: packages/sentinel_one/data_stream/threat/elasticsearch/ingest_pipeline/default.yml [hunk 1]
```diff
@@ -34,6 +34,17 @@ processors:
       formats:
           - ISO8601
       ignore_failure: true
+  - set:
+      field: event.action
+      copy_from: json.threatInfo.engines
```
Comment (andrewkroh) [inline packages/sentinel_one/data_stream/threat/elasticsearch/ingest_pipeline/default.yml:39]: This field appears to be an array and `event.action` should be a string. https://www.elastic.co/guide/en/ecs/current/ecs-event.html#field-event-action

File: packages/sentinel_one/data_stream/threat/elasticsearch/ingest_pipeline/default.yml [hunk 2]
```diff
@@ -34,6 +34,17 @@ processors:
...
+  - set:
+      field: user.name
```
Comment (andrewkroh) [inline packages/sentinel_one/data_stream/threat/elasticsearch/ingest_pipeline/default.yml:41]: `user.name` is set later on in the pipeline. In order to preserve the existing behavior I recommend to move this after the existing processor that sets it and add a conditional.

```
  - set:
      if: ctx.user?.name == null
      field: user.name
      copy_from: json.threatInfo.processUser
      ignore_empty_value: true
```

File: packages/sentinel_one/data_stream/threat/elasticsearch/ingest_pipeline/default.yml [hunk 3]
```diff
@@ -34,6 +34,17 @@ processors:
...
+      copy_from: json.threatInfo.processUser
+      if: ctx.json.threatInfo.processUser != null
+  - set:
+      field: process.name
+      copy_from: json.threatInfo.originatorProcess
+      if: ctx.json.threatInfo.originatorProcess != null
```
Comment (andrewkroh) [inline packages/sentinel_one/data_stream/threat/elasticsearch/ingest_pipeline/default.yml:47]: Instead of the `if` you can use `ignore_empty_value: true`.

**Review by andrewkroh — CHANGES_REQUESTED** [review]

**Review by ChriZzn — COMMENTED** [review]
Hi Andrew, 
thanks for the input, i have adjusted all problems/recommendations, should suit so far.

Regards Christoph

**Review by andrewkroh — APPROVED** [review]

**Pull request merged**